### ISSUE TITLE
HTTPNtlmAuthHandler throws an AttributeError in Python 3.4.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mullender/python-ntlm#46]
```
Using Python 3.4.1 with the latest version of the python30 directory as of July 
23, 2014. get_selector() and get_host() were deprecated in Python 3.3 and 
removed in 3.4, so HTTPNtlmAuthHandler throws an AttributeError. get_selector() 
should be replaced with request.selector and get_host() with request.host.

```

Original issue reported on code.google.com by `ge...@geoffblack.com` on 23 Jul 2014 at 8:37
- Blocked on: #48
- Merged into: #48
